+exp/SignalsExpTest.m

```diff
@@ -0,0 +1,436 @@
+classdef SignalsExpTest < handle
+  %SIGNALSEXPTEST Contains *Signals* information for running a *Signals* Exp Def from EXPTEST
+  %
+  % See also: EXPTEST, EXP.SIGNALSEXP
+  %
+  % Todo: set properties and methods attributes appropriately
+  
+  %% properties
+  properties %(Access = private)
+    
+    Clock = hw.ptb.Clock % 'Clock' object that returns current time (in s)
+    QuitKey = KbName('q') % Keyboard key for quitting experiment and closing 'ScreenH'
+    Net % 'sig.Net' object - signals network
+    T % 'sig.Node.OriginSignal' object - signals 'time' origin signal
+    Events % 'sig.Registry' object - signals 'events' ExpDef input arg
+    Params % 'sig.SubscriptableSignal' object - signals 'params' ExpDef input arg
+    VisStim % 'sig.StructRef' object - signals 'visStim' ExpDef input arg
+    Inputs % 'sig.Registry' object - signals 'inputs' ExpDef input arg
+    Outputs % 'sig.Registry' object - signals 'outputs' ExpDef input arg
+    Audio % 'audstream.Registry' object - signals 'audio' ExpDef input arg
+    ParamsLog % 'sig.SubscriptableSignal' object - log of parameters' time & value updates during experiment
+    Occ % handle to 'vis.init' shader - occulus visual stimuli viewing model
+    TexturesByID = containers.Map('KeyType', 'char', 'ValueType', 'uint32'); %'containers.Map' object - keys: texture ID strings; values: openGL ID
+    LayersByName = containers.Map %'containers.Map' object - visual element fields
+    ScreenH % handle to PTB Screen which displays visual stimuli
+    Data % 'struct' containing all the data from the experiment
+    Listeners % array of 'TidyHandle' listeners for signal updates
+    IsLooping = false; % flag for if experiment is running
+    StimWindowInvalid = false % flag for flipping PTB Screen (which must occur after VisStim changes)
+    CursorAsWheel = true % flag for having mouse cursor emulate steering wheel
+    CursorAsWheelKey = KbName('c') % keyboard key to set/unset 'CursorAsWheel'
+    CursorDelta = 0 % change in cursor position after disabling/enabling 'CurosorAsWheel'
+    CursorGain = 0.33 % gain factor for having cursor pixel X-location post into wheel
+    AsyncFlipping = false % flag for whether 'ScreenH' is currently asynchronously flipping
+    SignalUpdates = struct('name', cell(500,1), 'value', cell(500,1), 'timestamp', cell(500,1))
+    NumSignalUpdates = 0
+    GlobalPars % global parameters in GUI
+    CondPars % conditional parameters in GUI
+    
+    ExpTestObj % 'ExpTest' object - parent for this class (see constructor method)
```

Comment:
Ending properties and variable names with 'obj' is frowned upon: it's not important  which properties are object and which aren't and there are ways of determining the type (e.g. `class()`).  It's also inconsistent and many of these other properties are also objects.

Reply:
**thumbs up**